lighthouse-core/audits/seo/crawlable-anchors.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on whether anchors have hyperlinks which can be crawled by search engines. This descriptive title is shown when all hyperlinks on the page are crawlable. */
+  title: 'Anchors have crawlable hyperlinks',
+  /** Descriptive title of a Lighthouse audit that provides detail on whether anchors have hyperlinks which can be crawled by search engines. This descriptive title is shown when there are hyperlinks which are not crawlable by search engines. */
+  failureTitle: 'Anchors do not have crawlable hyperlinks',
+  /** Description of a Lighthouse audit that tells the user why hyperlinks should be crawlable. This is displayed after a user expands the section to see more. 'Learn More' becomes link text to additional documentation. */
+  description: 'Search engines use hyperlinks to crawl websites',
```

Comment:
we should flesh this out a bit more @AVGP any particular wording you'd like to see here?

maybe...

```suggestion
  description: 'Search engines use hyperlinks to crawl websites. Ensure that the `href` attribute of anchor elements links to an appropriate destination, so more pages of the site can be discovered. [Learn More](https://support.google.com/webmasters/answer/9112205)',
```

Reply:
I think this is great! Done. Won't resolve this yet in case there's any other ideas.

Reply:
Really good, indeed. Quick question: Are links okay in here or is that something we would put in the audit docs rather than here?

Reply:
we tend to have a single "Learn more" link that links to our web.dev doc on the issue and it's common for *that* article to link out to several more places. This felt like a nice placeholder link before we have the web.dev article available.